flake8_use_fstring/base.py

```diff
@@ -10,6 +12,18 @@
 
 Flake8Output = _typing.Tuple[_typing.Tuple[int, int], str]
 
+prefix_re = _re.compile(r"^[^'\"]*")
+
+
+def is_text_string_token(token: _TokenInfo) -> bool:
+    if not token.type == _token.STRING:
+        return False  # Not a string at all? Ignore.
+    # Get the prefix of the string (anything before the first quote)
+    prefix = prefix_re.match(token.string).group(0).lower()
+    if 'b' in prefix:
+        return False  # Smells like a bytestring. Ignore it.
+    return True
```

Comment:
It makes more sense to move these inside the class that's using them since they are not intended to be exposed.

Reply:
I moved it outside the module altogether since it can (and should) be used by prefix.py too.